### ISSUE TITLE
MONO_ENV_OPTIONS needs better documentation

### ISSUE THREAD

**Issue by mattjohnsonpint** [mono/website#585]
I stumbled across `MONO_ENV_OPTIONS` in [the 2.8 release notes](https://www.mono-project.com/docs/about-mono/releases/2.8.0/#mono_env_options).  I also see it mentioned once in the [LLVM docs page](https://www.mono-project.com/docs/advanced/mono-llvm/), but nowhere else.

This seems like a pretty fundamental feature.  Shouldn't it be highlighted somewhere more prominent?

In my case, I was able to set `MONO_ENV_OPTIONS="--debug"` so I can get filenames and line numbers when running tests under mono using `dotnet test` (when the test project targets .NET Framework and runs on Linux/macOS).

I found `mono --debug` in the [debugging docs](https://www.mono-project.com/docs/debug+profile/debug/#debugging-information), but couldn't figure out how to get it passed through with `dotnet test` until I found `MONO_ENV_OPTIONS`.

**Comment by lambdageek** [comment]
@akoeplinger I wonder if we can format the [mono manpage](https://github.com/mono/mono/blob/main/man/mono.1) and put it somewhere useful either on the mono website, or in the .NET docs.

Googling around, the online copies of the manpage are usually pretty out of date, and folks using modern .NET 6+ Mono don't install the MDK, so the manpage isn't available locally.

**Comment by mattjohnsonpint** [comment]
I didn't think to check the `man` page.  lol.

**Comment by akoeplinger** [comment]
Yeah I've been thinking about this as well, either the mono website or the .NET docs is fine with me.
